documentation/Working_on_Bot/README.md

```diff
@@ -0,0 +1,257 @@
+- [Test Cases](#test-cases)  
+- [Reporting Issues](https://github.com/CSSS/wall_e/wiki/4.-Reporting-Issues)  
+- [FAQs](#faqs)  
+
+
+## Running the Bot
+>If you encounter any errors doing the following commands, feel free to add it to the [FAQs section](documentation/Working_on_Bot#faqs) for future reference :)
+
+>Due to some compatibility issues that occured in the past when wall_e was dockerized only on the server, the dev environment for wall_e was changed so that each developer doesn't have to fight with OS issues that come with some python modules that are a bit iffy. as long as you can get `docker` and `docker-compose` working, you are golden to work on wall_e.
+
+Pre-requisites: `git`, [`docker`](https://docs.docker.com/install/linux/docker-ce/debian/#set-up-the-repository) and [`docker-compose`](https://docs.docker.com/compose/install/#install-compose)
+
+1. Fork the [Wall-e Repo](https://github.com/CSSS/wall_e.git)  
+2. clone the repo
+3. Wall_E Setting Specification.
+   1. Wall_e needs some settings in order to determine how some parts should be treated. The biggest component this impacts is whether or not you use wall_e with or without a dockerized database. [refer to the wiki page on the ini file](https://github.com/CSSS/wall_e/wiki/5.-contents-of-local.ini) for all the settings that wall_e reads from when doing local dev work.
+      1. Ways to specify settings: (please note that all the following options require the ini file with the same structure located [here](https://github.com/CSSS/wall_e/wiki/5.-contents-of-local.ini) to be placed at location `wall_e/src/resources/utilities/config/local.ini`. This is so that wall_e know what settings it will be taking in, even if the values in the ini file do not indicate the actual values it will take in.)
+         1. Specify via Env varibles.
+            1. Just export the settings [here](https://github.com/CSSS/wall_e/wiki/5.-contents-of-local.ini) with the specified values.
+         2. Specify via [`wall_e/src/resources/utilities/config/local.ini`](https://github.com/CSSS/wall_e/wiki/5.-contents-of-local.ini).
+            1. Be sure to not remove the headers on the ini. Also, please keep in mind that if you specify the same setting both via environment variable and via `.ini` file,  the environment variable will take precedence.
+         3. Via `CI/user_scripts/docker-compose-mount-nodb.yml` or `CI/user_scripts/docker-compose-mount.yml`.
+            1. Can be done following [these instructions](https://docs.docker.com/compose/environment-variables/#set-environment-variables-in-containers). Note that this is the same as via Env variables. The only difference is using this option will not result in the env variable be declared in your shell environment variable.
+         4. As you may see from the link in the previous point, docker provides multiple ways to pass variables. You can use any that work for you.
```

Comment:
Move this point out of the list of 3 ways to specify settings.

Reply:
don't really know where else to put it.

Reply:
>Be sure to not remove the headers on the ini.
Capitalize/Bold "not".

Reply:
>don't really know where else to put it.
Okay. Doesn't matter.

Reply:
>Capitalize/Bold "not".

fixed with c42e1e114bb5090ba55db89d828895907e85e7c4